C/plugins/utils/get_plugin_info.cpp

```diff
@@ -24,15 +24,20 @@ typedef PLUGIN_INFORMATION *(*func_t)();
  *
  * @param argv[1]  relative/absolute path to north/south C plugin shared library
  *
- * @param argv[2]  symbol to extract value from (typically 'plugin_info')
+ * @param argv[2]  symbol to extract value from (defaults 'plugin_info')
```

Comment:
So, it's optional now!

Reply:
Yes, it will not effect how we use it internally, but is easier for the user.